content/pt/docs/concepts/cluster-administration/certificates.md

```diff
@@ -0,0 +1,234 @@
+        ../cfssl print-defaults csr > csr.json
+1.  Crie um arquivo de configuração JSON para gerar o arquivo CA, por exemplo, `ca-config.json`:
+
+        {
+          "signing": {
+            "default": {
+              "expiry": "8760h"
+            },
+            "profiles": {
+              "kubernetes": {
+                "usages": [
+                  "signing",
+                  "key encipherment",
+                  "server auth",
+                  "client auth"
+                ],
+                "expiry": "8760h"
+              }
+            }
+          }
+        }
+1.  Crie um arquivo de configuração JSON para o CA certificate signing request (CSR), por exemplo,
+    `ca-csr.json`. Certifique-se de substituir os valores marcados com colchetes angulares por valores reais que você deseja usar.
+
+        {
+          "CN": "kubernetes",
+          "key": {
+            "algo": "rsa",
+            "size": 2048
+          },
+          "names":[{
+            "C": "<country>",
+            "ST": "<state>",
+            "L": "<city>",
+            "O": "<organization>",
+            "OU": "<organization unit>"
+          }]
+        }
+1.  Gere a chave CA (`ca-key.pem`) e o certificado (` ca.pem`):
+
+        ../cfssl gencert -initca ca-csr.json | ../cfssljson -bare ca
+1.  Crie um arquivo de configuração JSON para gerar chaves e certificados para o servidor de API, por exemplo, `server-csr.json`. Certifique-se de substituir os valores entre colchetes angulares por valores reais que você deseja usar. O `MASTER_CLUSTER_IP` é o cluster de serviço IP para o servidor da API, conforme descrito na subseção anterior. O exemplo abaixo também assume que você está usando `cluster.local` como DNS de domínio padrão
+
+        {
+          "CN": "kubernetes",
+          "hosts": [
+            "127.0.0.1",
+            "<MASTER_IP>",
+            "<MASTER_CLUSTER_IP>",
+            "kubernetes",
+            "kubernetes.default",
+            "kubernetes.default.svc",
+            "kubernetes.default.svc.cluster",
+            "kubernetes.default.svc.cluster.local"
+          ],
+          "key": {
+            "algo": "rsa",
+            "size": 2048
+          },
+          "names": [{
+            "C": "<country>",
+            "ST": "<state>",
+            "L": "<city>",
+            "O": "<organization>",
+            "OU": "<organization unit>"
+          }]
+        }
+1.  Gere a chave e o certificado para o servidor de API, que são, por padrão,
+    salvo no arquivo `server-key.pem` e` server.pem` respectivamente:
+
+        ../cfssl gencert -ca=ca.pem -ca-key=ca-key.pem \
+        --config=ca-config.json -profile=kubernetes \
+        server-csr.json | ../cfssljson -bare server
+
+
+## Distribuindo o Self-Signed CA Certificate
+
+Um nó cliente pode se recusar a reconhecer um self-signed CA certificate como válido.
+Para uma implementação de não produção ou para uma implementação executada por trás de um
```

Comment:
I would change 'implementação executada por trás de um firewall' to 'instalação que roda atrás de um firewall'